tests/playbooks/tests_bond_options.yml

```diff
@@ -124,17 +122,31 @@
                 interface_name: "{{ controller_device }}"
                 bond:
                   mode: active-backup
+                  arp_all_targets: any
```

Comment:
I think this should be "all" as "any" is the default value so we wouldn't notice even if NetworkManager is not setting it. Thank you!

Reply:
Nice catch.